### PR TITLE
Packager: fixes error during packing the app for MacOS.

### PR DESCRIPTION
**Pull request by rodrigoi**
During one of the version updates, the `asar` and `icns` filename changed. This is preventing the app to be built for MacOS (`make package-osx`).

From now on,`make package-osx` should produce a valid `.app` folder and `.dmg` packages.

### CODE REVIEW

File: builder.js [hunk 1]
```diff
@@ -50,21 +51,22 @@ var opts = {
...
+// 			ignore.push( path.join( directory, client[key] ) );
+// 		}
+// 	}
+//
+// 	return ignore;
+// }
+//
+// opts.ignore = opts.ignore.concat( whitelistInDirectory( './node_modules', [ 'deep-equal', 'electron-window-state', 'jsonfile', 'mkdirp' ] ) );
+// opts.ignore = opts.ignore.concat( whitelistInDirectory( './dist', [ 'index.html', 'app.js', 'manifest.appcache' ] ) );
+// opts.ignore = opts.ignore.concat( whitelistInDirectory( './', [ 'package.json', 'dist', 'desktop', 'node_modules' ] ) );
```
Comment (roundhill) [inline builder.js]: Should we just delete this commented out code, or do you think it will still be needed?

Comment (dmsnell) [inline builder.js]: I'm in favor of removing them, or fixing the reasons they are commented

**Comment by rodrigoi** [comment]
@roundhill @dmsnell I've updated the build script to package only the needed files and dependencies. Otherwise, everything is included on the packed release (including things on the `/releases` folder), potentially making the size of the distributable grow out of control.

To test, pack versions for Windows, macOS and linux. Executable versions should weight around 130mb, and `dmg`, `tar` and `exe` should weight around 45mb.

**Comment by roundhill** [comment]
> To test, pack versions for Windows, macOS and linux. Executable versions should weight around 130mb, and dmg, tar and exe should weight around 45mb.

Confirmed that this is the case :shipit:

**Pull request merged**